simplq/src/components/pages/Admin/QueueSettings.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import SettingsIcon from '@material-ui/icons/Settings';
+import Modal from 'components/common/Modal';
+import PrintIcon from '@material-ui/icons/Print';
+import HighlightOffIcon from '@material-ui/icons/HighlightOff';
+import { makeStyles } from '@material-ui/core/styles';
+import SidePanelItem from 'components/common/SidePanel/SidePanelItem';
+import { useUpdateQueueSettings } from 'store/asyncActions';
+import { useDispatch } from 'react-redux';
+import Grid from '@material-ui/core/Grid';
+import Button from '../../common/Button';
+import InputField from '../../common/InputField';
+
+const style = makeStyles((theme) => ({
+  modalContent: {
+    display: 'flex',
+    padding: theme.spacing(1),
+  },
+  actionContainer: {
+    display: 'flex',
+    justifyContent: 'space-evenly',
+  },
+  title: {
+    display: 'flex',
+    justifyContent: 'center',
+    textDecoration: 'underline',
+  },
+  sizeInput: {
+    marginBottom: theme.spacing(3),
+  },
+}));
+
```

Comment:
Could you move some of these to SCSS too?

Reply:
> > c. After clicking save, should I close the dialog
> 
> That sounds good.
Right now, I'm closing the dialog when user hits save. Is there a way to pass `toggleModal` as a callback to the action so that it will be called only in case of success?